lib/link_expansion/rules.rb

```diff
@@ -42,7 +42,8 @@ def recurring
     :withdrawn,
   ].freeze
 
-  CUSTOM_FIELDS_FOR_DOCUMENT_TYPE = {
+  CUSTOM_EXPANSION_FIELDS = {
```

Comment:
maybe these could be defined as hashes? ie
```
{ document_type: :redirect, fields: [] } 
```
then the code might do the talking rather than the comments? though downside is it could seem overly verbose?